docs/userguides/networks.md

```diff
@@ -365,11 +365,16 @@ To configure network URIs in `node`, you can use the `ape-config.yaml` file:
 
 ```yaml
 node:
+  # When managing or running a node, configure its IPC path directly (optional)
+  ipc_path: path/to/geth.ipc
```

Comment:
Does under here make sense?

Reply:
Do you mean the config path? It was there already from before, but yeah it's annoying it doesn't align with the others per-network.

We can add both.. Have this one serve as a global for all networks and the other one to be per network...

Reply:
4e8d2f3